src/datanode/src/alive_keeper.rs

```diff
@@ -409,147 +404,132 @@ impl CountdownTask {
                     }
                 }
                 () = &mut countdown => {
-                    let result = self.close_region().await;
-                    info!(
-                        "Region {region_id} is closed, result: {result:?}. \
-                        RegionAliveKeeper out.",
-                    );
-                    return Some(result);
+                    warn!("The region {region_id} lease is expired, set region to readonly.");
+                    let _ = self.region_server.set_writable(self.region_id, false);
+                    metrics::LEASE_EXPIRED_REGION
+                        .with_label_values(&[&format!("{}", self.region_id)])
+                        .inc();
+                    // resets the countdown.
+                    let far_future = Instant::now() + Duration::from_secs(86400 * 365 * 30);
+                    countdown.as_mut().reset(far_future);
```

Comment:
Is it possible to wrap it into an Option? Then we can set it to None instead of a far future magic number.